localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
 ﻿l_english:
+ ####################################################
+ # Special titles
+ ### Horde ###
+ warchief_male_emperor:0 "Warchief"
+ warchief_female_emperor:0 "Warchief"
 
-####################################################
-#Special titles
+ ### Scourge ###
+ lich_king_male:0 "Lich King"
+ lich_king_female:0 "Lich Queen"
+
+ ### Night Elves ###
  high_priest_kaldorei_male:0 "$HIGH_PRIEST_MALE_NAME$"
  high_priestess_kaldorei_female:0 "$HIGH_PRIEST_FEMALE_NAME$"
+
+ # Special cultural titles (sorted by culture, then government type).
+ ### Orcs ###
+ # Feudal
+ overlord_male_orc_count:0 "Overlord"
+ overlord_female_orc_count:0 "Overlord"
+ high_overlord_male_orc_duke:0 "High Overlord"
+ high_overlord_female_orc_duke:0 "High Overlord"
+ warlord_male_orc_king:0 "Warlord"
+ warlord_female_orc_king:0 "Warlord"
+ warchief_male_orc_emperor:0 "Warchief"
+ warchief_female_orc_emperor:0 "Warchief"
+
+ ### High Elves / Blood Elves ###
+ # Feudal
+ lord_male_highelf_count:0 "Lord"
+ lady_female_highelf_count:0 "Lady"
+ lordship_highelf_county:0 "Lordship"
+ high_king_male_highelf_emperor:0 "High King"
+ high_queen_female_highelf_emperor:0 "High Queen"
+ high_kingdom_highelf_empire:0 "High Kingdom"
+
```

Comment:
Honestly, i don't see why we shouldn't incorporate some flavor of the Highborne/Nightborne titles here. I'd say we name Kingdoms to Principality and Have Empire as Kingdom. thus we'd get King and Queen as empire titles, witch is thus equivalent to Queen Aszshara title, And i doubt any elf would give themself a greater title then here. I also think the Prince titles work well.

Reply:
Do you mean for kingdoms ?

Reply:
I am unsure on what you mean, can you be more specific on what you are wondering on?

Reply:
Realms remain realms because it is possible to lose the high realm of Quel'Thalas as for the date of Frozen Throne. So I think it should stay that way.

Reply:
That's a good point, but ain't that also the point?, when Anastarian died, Kael'thas had lost the high Kingdom, and did not take the title of King, but remained a prince. I think High Kingdom doesn't fit as well as Kingdom personally, Like if the realm split, i think it would be more flavor wise to have to prince's rather then 2 Kings in the region. What do you think about on that?

Reply:
It is purely a request from Kael'thas that he remain a prince and that his father be the last king of Quel'Thalas, it does not really represent the hierarchy of titles.
Moreover Kael'Thas is called the Sun King.
Humans kingdoms are also Kings, and humans emperors are High Kings.

Reply:
True, but there really have never been a "high" king/queen in warcraft. and i get what you mean but i think not using Prince's and Princesses we break from the Highborne/nightborne themes, and i think that High Elves are more closely cultural tied to them then they are the Humans on titles. But if this is something you really disagree with i will understand

Reply:
Yes they are called high kings. That's why we called them like this.
https://wowpedia.fandom.com/wiki/Anasterian_Sunstrider
For example

Reply:
Ok.. I am man enough to admit that i am wrong!